jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -92,8 +92,8 @@ YUI.add('environment-change-set', function(Y) {
       var cut;
       // If there is an object after the callback it's the
       // configuration object for ECS.
-      if (Y.Lang.isObject(lastParam) &&
-          Y.Lang.isFunction(args[args.length - 2])) {
+      if (typeof lastParam === 'object' &&
```

Comment:
This will have to expand to also have a not null check because `typeof null === 'object'` as well.

`typeof lastParam === 'object' && lastParam !== null && ... `